docs/source/examples/order_summary.rst

```diff
@@ -0,0 +1,225 @@
+
+Using this and the visualise tool, we can now choose better names for each of the fonts, and then load the document again, but this time providing a font mapping.
+
+.. code-block:: python
+
+   FONT_MAPPING = {
+       "BAAAAA+LiberationSerif-Bold,16.0": "title",
+       "BAAAAA+LiberationSerif-Bold,12.0": "sub_title",
+       "CAAAAA+LiberationSerif,12.0": "text",
+       "DAAAAA+FreeMonoBold,12.0": "table_header",
+       "EAAAAA+FreeMono,12.0": "table_text",
+   }
+   document = load_file("order_summary.pdf", font_mapping=FONT_MAPPING)
+
+Using the visualise tool again, we can now see that our element's font has changed to ``table_text``, which is a much more useful name for us.
+
+.. image:: /screenshots/order_summary_example/showing_font_2.png
+   :height: 300px
+
+Step 3 - Add sections
+.....................
+
+Another thing we can do to make our job easier is to add :class:`Sections<py_pdf_parser.sectioning.Section>` to our document. A :class:`Sections<py_pdf_parser.sectioning.Sectioning>` class is made available on :attr:`document.sectioning<py_pdf_parser.components.PDFDocument.sectioning>`, which in particular allows us to call :meth:`~py_pdf_parser.sectioning.Sectioning.create_section`.
+
+A section has a name, and contains all elements between the start element and the end element. You can add multiple sections with the same name, but each section will have both a ``name`` and a ``unique_name`` (which is just the name with an additional ``_n`` on the end, where ``n`` is the number of sections with that name).
+
+As with the :class:`~py_pdf_parser.components.PDFDocument`, a :class:`~py_pdf_parser.sectioning.Section` has an :attr:`~py_pdf_parser.sectioning.Section.elements` property which returns an :class:`~py_pdf_parser.filtering.ElementList`, allowing you to filter the elements.
+
+.. important:: Never instantiate a :class:`Sections<py_pdf_parser.sectioning.Section>` yourself. You should always use :meth:`~py_pdf_parser.sectioning.Sectioning.create_section`.
+
+Calling :meth:`~py_pdf_parser.sectioning.Sectioning.create_section` will return the :class:`~py_pdf_parser.sectioning.Section`, but the :class:`~py_pdf_parser.sectioning.Sectioning` class also has :meth:`~py_pdf_parser.sectioning.Sectioning.get_section` and :meth:`~py_pdf_parser.sectioning.Sectioning.get_sections_with_name` methods.
+
+Going back to our example, we will create sections for the order summary table, and for the totals table. Our order summary table will start with the "Order Summary:" sub title and end at the "Totals:" sub title. Note that there are two elements on the page with text equal to "Order Summary:", however they have different font and so we can still extract exactly the one we want.
+
+By default, meth:`~py_pdf_parser.sectioning.Sectioning.create_section` will include the last element in the section, but this can be disabled by passing ``include_last_element=False``.
```

Comment:
```suggestion
By default, :meth:`~py_pdf_parser.sectioning.Sectioning.create_section` will include the last element in the section, but this can be disabled by passing ``include_last_element=False``.
```